download/main.py

```diff
@@ -27,39 +20,35 @@ def unique_statuses(filers):
 
 def main():
     """ Do everyting """
-    data_dir_path = '.local/downloads'
-
     # pull data from gdrive and put it in .local/downloads
     pull_data(subfolder='main', default_folder='OpenDisclosure')
 
-    #engine = create_engine('postgresql+psycopg2://localhost/disclosure-backend-v2', echo=True)
-
-    with open(f'{data_dir_path}/elections.json', encoding='utf8') as f:
+    with open(f'{DATA_DIR_PATH}/elections.json', encoding='utf8') as f:
         elections_json = json.loads(f.read())
 
     elections = Elections(elections_json)
 
-    with open(f'{data_dir_path}/filers.json', encoding='utf8') as f:
+    with open(f'{DATA_DIR_PATH}/filers.json', encoding='utf8') as f:
         filers = json.loads(f.read())
 
-    committees = Committees.from_filers(filers, elections.df)
+    committees = Committees(filers, elections.pl)
 
     # A-Contribs:
     # join filers + filings + elections + transactions
     # transactions.filing_nid -> filings.filing_nid
     #   filings.filer_nid -> committees.filer_nid
     #     committees.Ballot_Measure_Election -> elections.Ballot_Measure_Election
     # where trans['transaction']['calTransactionType'] == 'F460A'
-    with open(f'{data_dir_path}/filings.json', encoding='utf8') as f:
-        filings = Filings(json.loads(f.read())).df
+    with open(f'{DATA_DIR_PATH}/filings.json', encoding='utf8') as f:
+        filings = Filings(json.loads(f.read())).pl
 
-    with open(f'{data_dir_path}/transactions.json', encoding='utf8') as f:
+    with open(f'{DATA_DIR_PATH}/transactions.json', encoding='utf8') as f:
         records = json.loads(f.read())
-        transactions = Transactions(records).df
+        transactions = Transactions(records).pl
 
-    a_contributions = A_Contributions(transactions, filings, committees.df)
+    a_contributions = A_Contributions(transactions, filings, committees.pl)
```

Comment:
To be consistent, perhaps `transactions` and `filings` should not be polars dataframes.  Also, perhaps A_Contributions should not be taking polars dataframes in the constructor since the idea is that you're abstracting the dataframes into objects and so you should be passing those objects around and only pull out the dataframes when you need them.